src/antares/model/study.py

```diff
@@ -216,6 +218,8 @@ def get_binding_constraints(self) -> MappingProxyType[str, BindingConstraint]:
     def create_area(
         self, area_name: str, *, properties: Optional[AreaProperties] = None, ui: Optional[AreaUi] = None
     ) -> Area:
+        if area_name in self._areas and isinstance(self._study_service, StudyLocalService):
```

Comment:
Should compare id and not name. And it should be done inside the local service method create_area, not here as you introduced the notion of local inside a model which isn't supposed to have one (plus it's a bit ugly)